frontend/src/components/pages/teacher/DisplayAssessmentResultsPage/index.tsx

```diff
@@ -61,6 +72,12 @@ const DisplayAssessmentResults = () => {
     skip: !!sessionTitle,
   });
 
+  const enableTeacherDataViz = checkFeatureFlag("ENABLE_TEACHER_DATA_VIZ");
```

Comment:
nit: 
```suggestion
  const isTeacherDataVizEnabled = checkFeatureFlag("ENABLE_TEACHER_DATA_VIZ");
```

(i.e from my understanding of the code, this should be a boolean right? not a function? the verb prefix makes it sound like a function)

Reply:
True